docs/components/form.html

```diff
@@ -73,6 +73,20 @@ <h1>Forms</h1>
 			</div>
 		</div>
 	</div>
+
+  <div class="container mt-4">
+    <h1>Newsletter Forms</h1>
```

Comment:
H1 must be used for page title not for this purpose. if you need the same size use the .h1 class